api/init.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	"sync"
+
+	"github.com/raintank/metrictank/expr"
+	"gopkg.in/raintank/schema.v1"
+)
+
+const defaultPointSliceSize = 2000
```

Comment:
The idea of the pool is very nice, but i think the limit of `2000` seems a little arbitrary. Would it make sense to make that based on some setting like f.e. `maxPointsPerReqHard`